mono/metadata/threads-types.h

```diff
@@ -382,7 +382,17 @@ MonoExceptionHandle
 mono_thread_interruption_checkpoint_handle (void);
 
 MonoException* mono_thread_force_interruption_checkpoint_noraise (void);
-gint32* mono_thread_interruption_request_flag (void);
+
+/**
+ * mono_thread_interruption_request_flag:
+ *
+ * Returns the address of a flag that will be non-zero if an interruption has
```

Comment:
comment is outdated

Reply:
will fix

Reply:
https://github.com/mono/mono/pull/17036.